test/e2e/semantic-domains.spec.ts

```diff
@@ -44,21 +44,11 @@ test.describe('Lexicon E2E Semantic Domains Lazy Load', () => {
   });
 
   test('Can change Project default language to Thai & back and forth', async () => {
```

Comment:
I chopped off the end of this test, because it seemed redundant (it still a bit redundant, but that's fine)